.github/ISSUE_TEMPLATE/2-feature_template.yml

```diff
@@ -0,0 +1,41 @@
+name: '💡 Feature Request'
+description: Create a new ticket for a new feature request
+title: '💡 [Feature Request] - <title>'
+labels: ['enhancement']
+
+body:
+  - type: textarea
+    id: summary
+    attributes:
+      label: 'Summary'
+      description: Provide a brief explanation of the feature
+      placeholder: Describe in a few lines your feature request
+    validations:
+      required: true
```

Comment:
I would add also a question :

**What problem are you trying to solve?**
*Tell us what is the thing you are doing and why this feature would help you in that*